sources/packages/backend/workflow/test/2022-2023/parttime-assessment/disbursements/parttime-assessment-disbursements.e2e-spec.ts

```diff
@@ -246,6 +280,295 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-disbursements`,
     },
   );
 
+  describe("Should generate 1 disbursement when offering weeks is greater than 17 weeks and ", () => {
```

Comment:
The 17-week logic is not the target of the test, right? We can just do the minimum to ensure one disbursement is generated and the numbers are properly rounded.
IMO the test description should be clear about the target of the test.
```
"Should round down the values for the disbursement when only one disbursement is expected and " +
    `the final award of each type is ${testFinalAwardNetAmount.finalAwardNetAmount}.` 
```